### ISSUE TITLE
error page for ftp login

### ISSUE THREAD

**Issue by lexoyo** [silexlabs/CloudExplorer2#33]
Right now when you try to login to ftp and give the wrong login or pass, the error is displayed as JSON
There should be a page to display auth errors for FTP
Then we should use this page also for SFTP and maybe other services too
CC @JbIPS

**Comment by JbIPS** [comment]
I gave this one some thoughts and the main reason the login page is in a separate window is because we were lazy.

We could totally set a redirectURI  for non-OAuth services to an endpoint that open a pop-in or something.

This way we stay in the React app and everything will be easier.

What do you think of that @lexoyo @SouroucheB?

**Comment by JbIPS** [comment]
Even better, we should check if a service use OAuth and use a different login strategy if it don't.

**Comment by jotakar** [comment]
interesting.
I've studing this option. I don't like a page to log FTP, I prefer popup window or similar.
Looking for the code I reached UnifileService.js in module cloud-explorer. There a window is open. Coul be this a good place to change the code?

**Comment by JbIPS** [comment]
Yes, that would be here to change the `window.open` by a new modal popin

**Comment by jotakar** [comment]
When ftp login fails the error window says that ok button allows repeat the login, but it open a blank window. The proccess loses the service name and can not open the right login window.
I've found how to create a modal login window, but I can not throw the error window (when login fails) in the same login window, any idea?
Thanks
